stubs/docker-compose.stub

```diff
@@ -3,11 +3,11 @@ version: '3'
 services:
     laravel.test:
         build:
-            context: ./vendor/laravel/sail/runtimes/8.0
+            context: ./vendor/laravel/sail/runtimes/8.1
```

Comment:
Dockerfile still counts on the dev version of PHP 8.1 `php8.1-dev`
Shouldn't we make sure that the Dockerfile is updated first?

Reply:
What do you mean? The https://github.com/laravel/sail/blob/1.x/runtimes/8.1/Dockerfile file is already updated.

Reply:
@driesvints Yup, this line https://github.com/laravel/sail/blob/31edbfef5c232157244564db4869fd93bc53b8ce/runtimes/8.1/Dockerfile#L24

Reply:
All of the Dockerfiles use the dev version.

Reply:
These pull in dev tools and are thus expected. Thanks